src/EnergyPlus/EvaporativeCoolers.cc

```diff
@@ -3270,16 +3270,17 @@ namespace EvaporativeCoolers {
             TEWB = EvapCond(EvapCoolNum).InletWetBulbTemp;
             TEDB = EvapCond(EvapCoolNum).InletTemp;
             InletNode = EvapCond(EvapCoolNum).InletNode;
-            if (EvapCond(EvapCoolNum).WetbulbEffecCurveIndex > 0) {
-                MassFlowRateSys = EvapCond(EvapCoolNum).InletMassFlowRate;
-                MassFlowRateSysDesign = Node(InletNode).MassFlowRateMax;
-                if (MassFlowRateSysDesign > 0.0) {
-                    if (MassFlowRateSys > 0.0) {
-                        FlowRatio = MassFlowRateSys / MassFlowRateSysDesign;
-                    } else {
-                        FlowRatio = 1.0;
-                    }
+
+            MassFlowRateSys = EvapCond(EvapCoolNum).InletMassFlowRate;
+            MassFlowRateSysDesign = Node(InletNode).MassFlowRateMax;
+            if (MassFlowRateSysDesign > 0.0) {
+                if (MassFlowRateSys > 0.0) {
+                    FlowRatio = MassFlowRateSys / MassFlowRateSysDesign;
+                } else {
+                    FlowRatio = 1.0;
                 }
+            }
+            if (EvapCond(EvapCoolNum).WetbulbEffecCurveIndex > 0) {
```

Comment:
I see, so the mass flow stuff should've just always been outside the curve index IF block.  You are still checking for divide by zero, but I don't see any handling of the MassFlowRateSysDesign == 0.0.  I realize this would mean things are going badly, but I want to make sure this case is at least gracefully handled and not going to divide by zero somewhere else down the line.

Reply:
OK, I checked it out.  FlowRatio is initialized properly to zero, and is used as a curve input variable below.  This is fine.